libnopegl/src/image.h

```diff
@@ -66,7 +66,7 @@ struct image_params {
 struct image {
     struct image_params params;
     struct texture *planes[4];
-    int nb_planes;
+    size_t nb_planes;
```

Comment:
`hwmap: change all nb_planes types to size_t` -> `image,hwmap: change all nb_planes types to size_t`